data-model/virtualcollections/v.0.1.0/schemas/virtualcollections.json

```diff
@@ -1 +1,138 @@
+                    "examples": [
+                        "Open to the public"
+                    ]
+                },
+                "ltc:collectionName": {
+                    "type": "string",
+                    "description": "https://tdwg.github.io/ltc/terms/index.html#ObjectGroup_collectionName A short title that summarizes the collection objects contained within the ObjectGroup.",
+                    "examples": [
+                        "Arctiacaja1991Netherlands"
+                    ]
+                },
+                "ltc:description": {
+                    "type": "string",
+                    "description": "https://tdwg.github.io/ltc/terms/index.html#ObjectGroup_description A free text description or narrative about the collection.",
+                    "examples": [
+                        "A virtual collection representing all butterfly specimens in South Holland"
+                    ]
+                }
+            }
+        },
+        "ltc:MeasurementOrFact": {
+            "type": "object",
+            "description": "https://tdwg.github.io/ltc/terms/index.html#MeasurementOrFact_MeasurementOrFact A measurement of or fact about a class within the standard, or a relationship between the ObjectGroup and an associated class.",
+            "properties": {
+                "dwc:measurementType": {
+                    "type": "string",
+                    "description": "https://tdwg.github.io/ltc/terms/index.html#MeasurementOrFact_measurementType The nature of the measurement, fact, characteristic, or assertion.",
+                    "examples": [
+                        "Object Count"
+                    ]
+                },
+                "dwc:measurementValue": {
+                    "type": "number",
+                    "description": "https://tdwg.github.io/ltc/terms/index.html#MeasurementOrFact_measurementValue The numerical value of the measurement.",
+                    "examples": [
+                        100
+                    ]
+                },
+                "dwc:measurementUnit": {
+                    "type": "string",
+                    "description": "https://tdwg.github.io/ltc/terms/#MeasurementOrFact_measurementUnit The units associated with the measurementValue.",
+                    "examples": [
+                        "mm, C, km, count, percentage, category"
+                    ]
+                }
+            }
+        },
+        "ods:searchFilters": {
+            "type": "array",
+            "description": "Search filters selected for the virtual collection.",
+            "items": {
+                "type": "object",
+                "properties": {
+                    "filterValue": {
+                        "type": "array",
+                        "examples": [
+                            "topicDiscipline=Zoology&midsLevel=1&country=Indonesia&hasMedia=1"
```

Comment:
This looks strange. I think we will probably change it to make sure what we need.
Not sure what the array is doing at the moment as everything is in 1 value.
I had more something like this in mind:
```
[ 
  { 
    "filterKey": "country",
    "filterValue": ["Netherlands", "Belgium"]
  },
  {
     "filterKey": "midsLevel",
     "filterValue": ["1"] // Keep filterValue an array of Strings
  }
]
```
So we would have an array of filterKey which is a string and filterValue which is an array of strings.